internal/k8sflags.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/util/homedir"
+)
+
+var defaultCacheDir = filepath.Join(homedir.HomeDir(), ".kube", "cache")
```

Comment:
If we don't have any of our own defaults, and only pass in non-zero flags, won't kubectl infer the default anyway? This would let us drop the client-go dep.

Reply:
Excellent idea, done.